compiler/extensions/doc/freemarker/overview.html.ftl

```diff
@@ -68,5 +68,16 @@
             (styleElementSheet.rules)? styleElementSheet.rules : styleElementSheet.cssRules;
         return styleElementRules[0].style;
       }
+
+    function getElementStyleFromID(styleItemId)
+    {
+        for (var styleItemId in allPackageNameListStyles)
+        {
+            var styleElementStyle = allPackageNameListStyles[styleItemId];
+            styleElementStyle.display = (event.data === "all" || styleItemId === event.data)? "list-item" : "none";
```

Comment:
Where does the event come from? I'm getting "TypeError: event is undefined" in Firefox.

Reply:
sorry, didn't replace the full function, fixed now